src/components/Genomes/Cobs/index.tsx

```diff
@@ -98,6 +101,60 @@ const CobsSearch: React.FC<CobsProps> = ({ catalogueName, catalogueID }) => {
       progress: undefined,
     });
 
+  const hypenateString = (str: string): string => {
+    const lowerCaseStr = str.toLowerCase();
+    const replacedSpaces = lowerCaseStr.replace(/ /g, '-');
+    return replacedSpaces.replace(/\./g, '-');
+  };
+
+  const getAccessionFromFirstGenome = async (): Promise<string> => {
+    const genomeName = hypenateString(catalogueName as string);
+    const url = `${config.api}genome-catalogues/${genomeName}/genomes?page=1&ordering=accession&page_size=1`;
+
+    try {
+      const response = await axios.get(url);
+      return response.data.data[0].attributes.accession;
+    } catch (responseErrors) {
+      return '';
+    }
+  };
+
+  const getSequenceCharsFromFastaFile = async (
+    fastaUrl: string
+  ): Promise<string> => {
+    try {
+      const headersConfig = {
+        headers: {
+          Range: 'bytes=0-499', // Fixed Range header value
+        },
+      };
+
+      const response = await axios.get(fastaUrl, headersConfig);
+      // Extract the DNA sequence from the data and get the first 50 characters
+      const { data } = response;
+      const lines = data.split('\n');
+      if (lines.length >= 2) {
+        const sequence = lines[1];
+        return sequence.substring(0, 50);
+      }
+      return '';
+    } catch (responseErrors) {
+      return '';
+    }
+  };
+
+  useEffect(() => {
+    const buildSequence = async () => {
+      const myAccession = await getAccessionFromFirstGenome();
+      const fastaUrl = `${config.api}genomes/${myAccession}/downloads/${myAccession}.fna`;
+      const firstFiftySequenceChars = await getSequenceCharsFromFastaFile(
+        fastaUrl
+      );
+      textareaSeq.current.quill.setText(firstFiftySequenceChars);
```

Comment:
Do we need a spinner in the text area? Depending on the user network.. the sequence will randomly appear at any time or am I mistaken?.